testcontainers/src/images/local_stack.rs

```diff
@@ -0,0 +1,72 @@
+use crate::{core::WaitFor, Image};
+use std::collections::HashMap;
+
+const NAME: &str = "localstack/localstack";
+const TAG: &str = "latest";
+const PORT: u16 = 4566;
+
+#[derive(Debug)]
+pub struct LocalStack {
+    env_vars: HashMap<String, String>,
+    volumes: HashMap<String, String>,
+    tag: String,
+}
+
+impl LocalStack{
+    pub fn set_services(&mut self, services: &str) {
+
+        self.env_vars.insert("SERVICES".to_owned(), services.to_owned());
+
+    }
```

Comment:
Having an API to configure the services seems unnecessary for the latest versions of LocalStack. The services are "available" immediately anyway.

For Testcontainers implementations that do not need to support backwards compatibility (older versions) like .NET and Go we decided not to add an API to configure the services. Developers can simply use the API that configures environment variables (if necessary).

Reply:
ok